### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by MattKunze**
Merge changes from upstream project

### CODE REVIEW

**Comment by davidhaimson** [comment]
Can't merge this now because there are some problems with this version of the fixed-data-table-2 code:

[Column resize problems](https://github.com/schrodinger/fixed-data-table-2/issues/276)
[Column resize doesn't work on Android](https://github.com/schrodinger/fixed-data-table-2/issues/277)

File: src/FixedDataTableCell.js [hunk 1]
```diff
@@ -246,11 +251,18 @@ var FixedDataTableCell = createReactClass({
       var columnResizerStyle = {
         height
       };
+      function suppress(event) {
```
Comment (davidhaimson) [inline src/FixedDataTableCell.js]: All my changes are from here (line 254)...

File: src/FixedDataTableCell.js [hunk 2]
```diff
@@ -317,6 +331,14 @@ var FixedDataTableCell = createReactClass({
...
       event
     );
+    /**
+     * This prevents the rows from moving around when we resize the
+     * headers on touch devices.
+     */
+    if (this.props.touchEnabled) {
+      event.preventDefault();
+      event.stopPropagation();
+    }
```
Comment (davidhaimson) [inline src/FixedDataTableCell.js]: ... to here (line 341).

**Comment by davidhaimson** [comment]
I merged Version 0.8.9 (fix for #276) and made the necessary changes to FixedDataTableCell.js to make it work on Android.

Tested and works correctly on:
- Chrome/Android
- Cordova/Android
- Safari/iOS11
- Cordova/iOS11
- Chrome/Mac
- Safari/Mac
- Firefox/Mac
- IE11/Win7
- Edge/Win10

When resizing columns, the handle (4px wide) is just to the left of the column header cell border. If you drag within the header (and do not wander up or down out of the header), it works great. If you do wander up or down, it fails. This is because of a design flaw (bug) is in an external Facebook library that is not interested in accepting PRs. I will point this out to the people in the Schrodinger project.

**Comment by davidhaimson** [comment]
I can no longer reproduce the "wander up or down" bug, so never mind.

**Comment by davidhaimson** [comment]
Fixes https://github.com/DataSplice/datasplice-client/issues/2271

**Comment by MattKunze** [comment]
I'm reorganizing this so the upstream merge and resize fixes are separate branches

**Pull request merged**